macros/drop_schemas.sql

```diff
@@ -0,0 +1,31 @@
+{% macro drop_schemas(drop_target_schema=true) %}
+
```

Comment:
Should we add the dispatch config to this macro as well? Additionally what are your thoughts on naming this something unique to ensure there is no overlap with matching macro names in users projects?

Reply:
yeah i think we should. added both a dispatch and different name